Plugin.Maui.Biometric/Plugin.Maui.Biometric/Shared/BiometricType.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+namespace Plugin.Maui.Biometric.Shared
```

Comment:
@robertmaxted Can you please remove Shared from the namespace...

Reply:
@FreakyAli I think that is due to it all being in develop, can you lift and shift all of the logic into a new branch based on the correct branch (master ? ).

Reply:
done